resources/public/api/conf/1.0/application.yaml

```
@@ -19,14 +19,14 @@ servers:
 
 components:
   securitySchemes:
-    User-Restricted:
+    Application-Restricted:
       type: oauth2
-      description: HMRC supports OAuth 2.0 for authenticating [User-restricted](https://developer.qa.tax.service.gov.uk/api-documentation/docs/authorisation/user-restricted-endpoints) API requests
+      description: |
+        HMRC supports OAuth 2.0 for authenticating [Application-Restricted](https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/application-restricted-endpoints) API requests using an OAuth 2.0 Bearer Token
+        in the AUTHORIZATION header.
```

We don't normally specify that the OAuth 2.0 Bearer Token needs to be in the AUTHORIZATION header in other OAS docs. Doing so is arguably superfluous as a vendor can click on the link for an explanation of how to use the OAuth token
```suggestion
        HMRC supports OAuth 2.0 for authenticating [Application-Restricted](https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/application-restricted-endpoints) API requests.
```